inputs/BaseInput.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace GameOfLife\inputs;
+
+use GameOfLife\Field;
+use Ulrichsg\Getopt;
+
+/**
+ * Base input
+ * With the function addOptions, you can add Options from outside of the gameoflife.php file.
+ * The function fillField can be used to fill the $field variable in Field.php with the wished field-stats.
+ * @package GameOfLife\inputs
```

Comment:
You can leave this here for your project, but in DiLoc|Motion we do not use the `@package`-notation.